site/CookiePreferencesManager.tsx

```diff
@@ -26,7 +26,7 @@ export interface Preference {
 
 export const POLICY_DATE: number = 20201009
 export const DATE_FORMAT = "YYYYMMDD"
-const COOKIE_NAME = "cookie_preferences"
+export const COOKIE_NAME = "cookie_preferences"
```

Comment:
Do we want to rename this to something like `COOKIE_PREFERENCES_COOKIE_NAME`, and the other one to `DATA_INSIGHTS_NEWSLETTER_BANNER_COOKIE_NAME` (it's long, I know)?

That way, there'll never be any confusion anywhere.

Reply:
The length is painful, but it is for the better. Changed.